### ISSUE TITLE
sslint: cannot find module umd.js

### ISSUE THREAD

**Issue by nilleb** [anubra266/choc-autocomplete#212]
$ eslint src --ext ts,tsx --report-unused-disable-directives --max-warnings 0 --fix

Oops! Something went wrong! :(

ESLint: 8.49.0

Error: Cannot find module '/Users/nilleb/dev/nillebco/adb-expertise-locator-ui/node_modules/@choc-ui/chakra-autocomplete/dist/index.umd.js'
Occurred while linting /Users/nilleb/dev/nillebco/adb-expertise-locator-ui/src/pages/profile/page.tsx:1
Rule: "import/namespace"
    at createEsmNotFoundErr (node:internal/modules/cjs/loader:1022:15)
    at finalizeEsmResolution (node:internal/modules/cjs/loader:1015:15)
    at resolveExports (node:internal/modules/cjs/loader:529:14)
    at Function.Module._findPath (node:internal/modules/cjs/loader:569:31)
    at findModulePath (/Users/nilleb/dev/nillebco/adb-expertise-locator-ui/node_modules/eslint-import-resolver-alias/index.js:99:27)
    at Object.exports.resolve (/Users/nilleb/dev/nillebco/adb-expertise-locator-ui/node_modules/eslint-import-resolver-alias/index.js:75:10)
    at withResolver (/Users/nilleb/dev/nillebco/adb-expertise-locator-ui/node_modules/eslint-module-utils/resolve.js:114:23)
    at fullResolve (/Users/nilleb/dev/nillebco/adb-expertise-locator-ui/node_modules/eslint-module-utils/resolve.js:135:22)
    at Function.relative (/Users/nilleb/dev/nillebco/adb-expertise-locator-ui/node_modules/eslint-module-utils/resolve.js:84:10)
    at remotePath (/Users/nilleb/dev/nillebco/adb-expertise-locator-ui/node_modules/eslint-plugin-import/lib/ExportMap.js:811:381)
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

**Comment by Kysluss** [comment]
Hi @nilleb

Can you post your eslint config?  Running on windows from a base installation, I don't get this error.  Once I can reproduce, I'll get it fixed.

**Comment by nilleb** [comment]
Yeah, sure! Thanks a lot for having a look at this!
```
{
    "root": true,
    "env": {
        "node": true,
        "es6": true,
        "jest": true
    },
    "parserOptions": {
        "ecmaVersion": 8
    },
    "ignorePatterns": [
        "node_modules/*",
        ".out/*",
        "!.prettierrc.js",
        ".d.ts"
    ],
    "settings": {
        "react": {
            "version": "detect"
        },
        "import/resolver": {
            "alias": {
                "map": [
                    [
                        "@",
                        "./src"
                    ]
                ],
                "extensions": [
                    ".ts",
                    ".tsx"
                ]
            }
        }
    },
    "plugins": [
        "@typescript-eslint"
    ],
    "extends": [
        "eslint:recommended",
        "plugin:react/recommended",
        "plugin:import/recommended",
        "plugin:jsx-a11y/recommended",
        "plugin:import/typescript",
        "plugin:@typescript-eslint/recommended",
        "plugin:prettier/recommended",
        "prettier",
        "eslint-config-prettier"
    ],
    "rules": {
        "import/no-anonymous-default-export": "off",
        "@typescript-eslint/explicit-module-boundary-types": "off",
        "prefer-const": "error",
        "@typescript-eslint/no-unused-vars": "error",
        "@typescript-eslint/no-explicit-any": "off",
        "react/prop-types": "off",
        "react/react-in-jsx-scope": "off",
        "jsx-a11y/anchor-is-valid": "off",
        "@typescript-eslint/explicit-function-return-type": "off",
        "prettier/prettier": [
            "error",
            {},
            {
                "usePrettierrc": true
            }
        ],
        "indent": [
            "error",
            2,
            {
                "SwitchCase": 1
            }
        ]
    },
    "overrides": [
        {
            "files": [
                "**/*.stories.jsx",
                "**/*.jsx"
            ],
            "rules": {
                "import/no-anonymous-default-export": "off",
                "@typescript-eslint/explicit-module-boundary-types": "off"
            }
        }
    ]
}
```

**Comment by nilleb** [comment]
The package.json looks like this
```
{
  "name": "adb-expertise-locator-ui",
  "private": true,
  "license": "UNLICENSED",
  "version": "0.0.0",
  "type": "module",
  "scripts": {
    "dev": "vite",
    "build": "tsc && vite build",
    "lint": "eslint src --ext ts,tsx --report-unused-disable-directives --max-warnings 0 --fix",
    "preview": "vite preview",
    "storybook": "storybook dev -p 6006",
    "build-storybook": "storybook build"
  },
  "dependencies": {
    "@ariakit/react": "^0.2.17",
    "@azure/msal-browser": "^2.38.0",
    "@azure/msal-react": "^1.5.9",
    "@chakra-ui/icons": "^2.0.19",
    "@chakra-ui/react": "^2.7.0",
    "@choc-ui/chakra-autocomplete": "^5.2.5",
    "@datadog/browser-rum": "^4.46.0",
    "@emotion/react": "^11.11.1",
    "@emotion/styled": "^11.11.0",
    "@tanstack/react-query": "^4.29.23",
    "axios": "^1.4.0",
    "date-fns": "^2.30.0",
    "framer-motion": "^10.12.16",
    "match-sorter": "^6.3.1",
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "react-hotkeys-hook": "^4.4.1",
    "react-nanny": "^2.15.0",
    "react-router-dom": "^6.12.1",
    "react-use": "^17.4.0"
  },
  "devDependencies": {
    "@chakra-ui/storybook-addon": "^5.0.1",
    "@storybook/addon-essentials": "^7.1.1",
    "@storybook/addon-interactions": "^7.1.1",
    "@storybook/addon-links": "^7.1.1",
    "@storybook/blocks": "^7.1.1",
    "@storybook/react": "^7.1.1",
    "@storybook/react-vite": "^7.1.1",
    "@storybook/testing-library": "^0.2.0",
    "@types/react": "^18.0.37",
    "@types/react-dom": "^18.0.11",
    "@types/react-highlight-words": "^0.16.4",
    "@typescript-eslint/eslint-plugin": "^5.13.0",
    "@typescript-eslint/parser": "^5.0.0",
    "@vitejs/plugin-react": "^4.0.0",
    "eslint": "^8.49.0",
    "eslint-config-airbnb": "^19.0.4",
    "eslint-config-airbnb-typescript": "^17.0.0",
    "eslint-config-prettier": "^9.0.0",
    "eslint-config-react-app": "^7.0.1",
    "eslint-config-standard-with-typescript": "^36.0.0",
    "eslint-import-resolver-alias": "^1.1.2",
    "eslint-import-resolver-typescript": "^3.6.0",
    "eslint-import-resolver-vite": "^1.3.0",
    "eslint-plugin-import": "^2.25.2",
    "eslint-plugin-n": "^15.0.0",
    "eslint-plugin-prettier": "^5.0.0",
    "eslint-plugin-promise": "^6.0.0",
    "eslint-plugin-react": "^7.32.2",
    "eslint-plugin-react-hooks": "^4.6.0",
    "eslint-plugin-react-refresh": "^0.3.4",
    "eslint-plugin-storybook": "^0.6.13",
    "msw": "^1.2.3",
    "msw-storybook-addon": "^1.8.0",
    "prop-types": "^15.8.1",
    "storybook": "^7.1.1",
    "storybook-addon-react-router-v6": "^2.0.3",
    "typescript": "*",
    "vite": "^4.3.9",
    "vite-plugin-eslint": "^1.8.1"
  },
  "msw": {
    "workerDirectory": "public"
  }
}
```
(I am sorry I can not disclose access to the sources)
(the profile/page.tsx doesn't even include the package, it's another component which does)

**Comment by Kysluss** [comment]
Thanks @nilleb!  I was able to reproduce this tonight with the config you provided, thank you.  I think this is fixed in [v5.2.7](https://github.com/anubra266/choc-autocomplete/releases/tag/v5.2.7).  Let me know if this fixes it or if it still happens.

**Comment by nilleb** [comment]
Thanks a kit @Kysluss - I do confirm that this is fixed. Many thanks for your prompt intervention, kudos to you 🙇‍♂️
